marimo/_plugins/stateless/image.py

```diff
@@ -23,24 +23,30 @@ def image(
 ) -> Html:
     """Render an image as HTML.
 
-    **Example.**
+    **Examples.**
 
     ```python3
+    # Render an image from a url
     mo.image(
         src="https://marimo.io/logo.png",
         alt="Marimo logo",
         width=100,
         height=100,
         rounded=True,
     )
+    ```
 
+    ```python3
+    # Render an image from a local file
     with open("logo.png", "rb") as file:
-        mo.image(src=file)
+        image = mo.image(src=file)
```

Comment:
any downside to auto-opening when the file is a local path?

Reply:
Nope! I think we should do that. Just need to write some logic to detect if it's a URL or not.

Reply:
Can leave for another PR? Since we'll keep the file support anyway.

Reply:
i guess, wouldnt we want to do the heavy lifting for the user? seems like a suboptimal example otherwise?

Reply:
done